README.md

```diff
@@ -1,2 +1,22 @@
 # pushito
-APNS over HTTP/2
+APNS over HTTP/2 for Elixir. Inspired on Erlang the project [apns4erl](https://github.com/inaka/apns4erl)
+
+**note** this project is under development
+
+## Tests
```

Comment:
You should consider to create an integration test folder to do this – Check [Ecto](https://github.com/elixir-ecto/ecto)